custom-elements/state.tentative/ElementInternals-states.html

```diff
@@ -26,9 +26,9 @@
 
 test(() => {
   let i = (new TestElement()).internals;
-  assert_throws(new TypeError(), () => { i.states.supports('foo'); });
-  assert_throws(new SyntaxError(), () => { i.states.add(''); });
-  assert_throws('InvalidCharacterError', () => { i.states.add('a\tb'); });
+  assert_throws_js(TypeError, () => { i.states.supports('foo'); });
+  assert_throws_dom('SyntaxError', () => { i.states.add(''); });
```

Comment:
It might be worth pointing out that this also fixes correctness of test as `new SyntaxError()` seems wrong. (Same for some other files.)

Reply:
Pointing out in the commit message?  But yes, in general that was the whole point of the assert_throws changes...

Reply:
I thought the main point was asserting the correct global, not also changing some incorrect JS SyntaxError to "SyntaxError" DOMException.

Reply:
Fwiw, the main point for _me_ when I started this was catching places where people threw bogus DOMException TypeErrors instead of JS TypeErrors.  The JS-vs-DOM SyntaxError is basically the same thing.

Reply:
Oh, I didn't realize the state of affairs was that bad. Wow, well I'm even more glad you fixed this then, thanks so much!